portfolio/src/main/webapp/script.js

```diff
@@ -129,13 +129,27 @@ function getComment() {
  * @return {element}
  */
 function createCommentElement(comment) {
+  // Create element for comment.
   const commentElement = document.createElement('li');
   commentElement.className = 'comment';
 
+  // Create element for content and append it to comment.
   const contentElement = document.createElement('span');
   contentElement.innerText = comment.content;
-
   commentElement.appendChild(contentElement);
+
+  // Create element for image if it exists and append it to comment.
+  if (comment.imageUrl != null) {
+    const imageSourceElement = document.createElement('img');
+    imageSourceElement.src = comment.imageUrl;
+    const imageAnchorElement = document.createElement('a');
+    imageAnchorElement.href = comment.imageUrl;
```

Comment:
Since this is an image, we'll want to include an alt tag here for accessibility purposes! 

This one is tricky to catch but since you're forming html within this js file, you'll want to keep in mind the html style guide: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

Reply:
Oh, nice catch! Wouldn't have thought about it inside the JS. Done!